src/control-plane/backend/lambda/api/common/explore-types.ts

```diff
@@ -87,10 +87,10 @@ export enum ExplorePathSessionDef {
 
 export enum ExplorePathNodeType {
   EVENT = 'event',
-  PAGE_TITLE = '_page_title',
-  PAGE_URL = '_page_url',
-  SCREEN_NAME = '_screen_name',
-  SCREEN_ID = '_screen_id',
+  PAGE_TITLE = 'page_title',
```

Comment:
I see the duplicated `ExplorePathNodeType` in `frontend/src/ts/explore-types.ts`.

We need extract them to base lib.